models/data_warehouse.model.lkml

```diff
@@ -3957,7 +3957,7 @@ explore: performance_events {
 
 explore: 1day_retention_rate {
   from: 1day_retention_rate
-  label: " Growth Scorecard 1-day Retention Rate"
+  label: "1-day Retention Rate"
```

Comment:
```suggestion
  label: "Day 1 Retention Rate"
```

The technical term is Day 1 Retention. 1-day retention measures it slightly differently. I'm sorry I just realized this, would it be too much effort to change the explore name to `day1_retention_rate` as well?

Reply:
Updated the labels and titles. Keeping dbt stuff as is, for consistency.